model/vm.rb

```diff
@@ -163,6 +163,11 @@ def inhost_name
     self.class.ubid_to_name(UBID.from_uuidish(id))
   end
 
+  def vm_host_family
```

Comment:
I'd probably name this `host_family`

Then calls to this would look like `vm.host_family` instead of `vm.vm_host_family`.

Reply:
I removed this helper and kept `family_filter = ["standard"] if vm.family == "burstable"` line